OracleManagementAgent/dockerfiles/latest/Dockerfile

```diff
@@ -0,0 +1,57 @@
+# Copyright (c) 2022 Oracle and/or its affiliates. All rights reserved.
+#
+# Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+#
+# ORACLE DOCKERFILES PROJECT
+# --------------------------
+# This is the Dockerfile for Oracle Management Agent
+#
+# Base image of this dockerfile is Oracle Linux 7 Slim docker image.
+# 
+# REQUIRED FILES TO BUILD THIS IMAGE
+# ----------------------------------
+# (1) Management Agent software for Linux (Linux-x86_64/latest/oracle.mgmt_agent.zip)
+#
+# How to download the Management Agent software:
+# https://docs.oracle.com/en-us/iaas/management-agents/doc/install-management-agent-chapter.html
+# 
+# HOW TO BUILD THIS IMAGE
+# -----------------------
+# Put all downloaded files in the same directory as this Dockerfile
+# Run:
+#      $ docker build -t oracle/mgmtagent-container .
+#
+# This command is already scripted in build.sh so you can alternatively run
+#		$ bash build.sh
+#
+
+FROM oraclelinux:7-slim as builder
+
+MAINTAINER Drupad Panchal <drupad.panchal@oracle.com>
```

Comment:
The `MAINTAINER` directive is deprecated and should not be used.

Reply:
fixed